nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-0-10-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/ConsumeKafkaRecord_0_10.java

```diff
@@ -160,6 +163,11 @@
             .name("success")
             .description("FlowFiles received from Kafka.  Depending on demarcation strategy it is a flow file per message or a bundle of messages grouped by topic and partition.")
             .build();
+    static final Relationship REL_PARSE_FAILURE = new Relationship.Builder()
```

Comment:
@markap14 this is good but i think we'll need the metadata (topic,partition,offset) as that would obviously be helpful to folks troubleshooting such issues.